### DISCUSSION TITLE
DietPi-Software | Gitlab

### DISCUSSION THREAD

**Discussion by rafidalhaque** [MichaIng/DietPi#5438] [Software Request]
#### Formal software information
- Software title | Gitlab
- Short description | Github open source alternative
- Official URL (if available) |  https://gitlab.com 
- GitHub URL (if available) | 
- (Official) install documentation | 

#### Are there similar/alternative software titles available with DietPi-Software?

- Gitea, Gogs

#### What makes your requested software better than the above solutions, if available?

- It is open source. Too much famous. And also have some unique features better than gitea or gogs.

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?

- ...

#### Can you provide the installation steps that you would suggest DietPi-Software to do?

1. ...

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.

- Sorry, I'm not very expert to maintain it 😪

**Comment by Joulinar** [comment]
Might be similar to #500 right?

**Reply by rafidalhaque** [reply]
Right 😃

**Reply by rafidalhaque** [reply]
But couldn't gitlab officially implement in dietpi?

Issue #500 didn't help me in that question.

**Reply by Joulinar** [reply]
The topic has been closed as there was no time to implement and it was at low interest.

**Reply by rafidalhaque** [reply]
😥

**Reply by Joulinar** [reply]
I moved your request to discussions where people could vote for it now.

**Reply by rafidalhaque** [reply]
Thanks :smiley:
